frontend/components/Dashboard/Posts/Post.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react'
+import Link from 'next/link'
+import { useTranslation } from '../../../config/i18n'
+import { formatShortDate } from '../../../utils/date'
+import { Post as PostType } from '../../../generated/graphql'
+import XIcon from '../../Icons/XIcon'
+import theme from '../../../theme'
+
+type Props = {
+  post: PostType
+  avatar?: boolean
+}
+
+const postBorderRadius = '5px'
+
+const Post: React.FC<Props> = ({ post, avatar = false }) => {
+  const { t } = useTranslation('posts')
+  const {
+    id,
+    title,
+    excerpt,
+    readTime,
+    images,
+    likes,
+    threads,
+    author: { handle, name, profileImage },
+    createdAt,
+  } = post
+
+  const displayImage = images.length ? images[0].smallSize : '/images/samples/sample-post-img.jpg'
+  console.log(handle, name, profileImage, avatar)
```

Comment:
`console.log`

Reply:
This is just to get rid of the TypeScript errors for unused variables until the avatar part is implemented in the next PR 😁